docs/installation/installing-with-npm.md

```diff
@@ -1,303 +1,196 @@
 
-```JS
-import { Radios } from 'govuk-frontend'
-```
+### Add the JavaScript file to your HTML
 
-If you're using a bundler such as [Browserify](http://browserify.org/), you may need to use the CommonJS `require`:
+First either:
 
+- route requests to the `node_modules/govuk-frontend/govuk/all.js` file
+- copy the file into your project folder
 
-```JS
-const GOVUKFrontend = require('govuk-frontend')
+Then add GOV.UK Frontend's Javascript file before the closing `</body>` tag of your HTML page or page template, and run the `initAll` function to initialise all the components.
 
-const Radios = GOVUKFrontend.Radios
+```html
+<body>
+...
+  <script src="node_modules/govuk-frontend/govuk/all.js"></script>
+  <script>
+    window.GOVUKFrontend.initAll()
+  </script>
+</body>
 ```
 
-GOV.UK Frontend components with JavaScript behaviour have the `data-module` attribute set in their markup.
+You can also select specific components, then call `initAll` with a `scope` parameter that includes only those components. For example, to initialise only [radios](https://design-system.service.gov.uk/components/radios/):
 
-You can use this attribute to initialise the component manually, this may be useful if you are adding markup to a page after it has loaded.
-
-To initialise the first radio component on a page, use:
-
-```js
-var $radio = document.querySelector('[data-module="govuk-radios"]')
-if ($radio) {
-  new Radios($radio).init()
-}
+```html
+...
+  <script>
+    var radio = document.querySelector('.radio')
+    window.GOVUKFrontend.initAll({
+      scope: $radio
+    })
+  </script>
 ```
 
-### Polyfills
-A JavaScript polyfill provides functionality on older browsers or assistive technology that do not natively support it.
-
-The polyfills provided with GOV.UK Frontend aim to fix usability and accessibility issues. If there is a JavaScript included in the component directory, it is important to import and initialise it in your project to ensure that all users can properly use the component (see [Polyfilling](/docs/contributing/polyfilling.md)).
+If you add markup to a page after it's loaded, you can also use the `data-module` attribute to select specific components. For example, you can select radios using their `data-module` attribute of `govuk-radios`.
 
-### How GOV.UK Frontend is bundled
-The JavaScript included in GOV.UK Frontend components are in [UMD (Universal Module Definition)](https://github.com/umdjs/umd) format which makes it compatible with AMD (Asynchronous module definition) and CommonJS.
+#### Import JavaScript using a bundler
 
-See [JavaScript Coding Standards](/docs/contributing/coding-standards/js.md) for more details of how JavaScript is used in the project.
+Use `import` to import all of GOV.UK Frontend's components, then run the `initAll` function to initialise them:
 
-#### Using GOV.UK Frontend with Webpack 4
-Here's an example of setting up [`webpack.config.js`](examples/webpack/webpack.config.js) in your project
```

Comment:
I think this is the only place from where we link the Webpack config file. Should we mention it somewhere else instead?

Reply:
This was kept at one stage, but @36degrees felt it was best to remove it during an earlier review.